### PR TITLE
Add populate examples.

### PR DESCRIPTION
**Pull request by eddyystop**

### CODE REVIEW

File: api/hooks-common.md [hunk 1]
```diff
@@ -491,6 +491,90 @@ Populates items *recursively* to any depth. Supports 1:1, 1:n and n:1 relationsh
...
+- 1:1 relationship
+
+```javascript
+// users like { _id: '111', name: 'John', roleId: '555' }
+// roles like { _id: '555', permissions: ['foo', bar'] }
+import { populate } from 'feathers-hooks-common';
+
+const userRoleSchema = {
+  include: {
+    service: roles,
```
Comment (hubgit) [inline api/hooks-common.md]: The examples as currently written will fail, as the values are missing single quotes.

File: guides/step-by-step/intro/not-worry.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ These are covered in the companion guides.
 ## Is Feathers production ready? Is it scalable?
 
 Yes, and yes.
-There's detailed information [below](../production-ready.md).
+There's detailed information in [this post.](https://www.quora.com/Is-FeathersJS-production-ready)
```
Comment (hubgit) [inline guides/step-by-step/intro/not-worry.md]: This seems to be an unrelated edit?

Comment (marshallswain) [inline guides/step-by-step/intro/not-worry.md]: Yes.  Please don't pick on us.  ;)

File: api/hooks-common.md (hunk 1 above)
Comment (marshallswain) [inline api/hooks-common.md]: PR welcome.  ;)

**Comment by eddyystop** [comment]
Thanks @Alf Eaton.

I had a chuckle over your comment @marshall.

On Sat, Apr 15, 2017 at 5:41 PM, Marshall Thompson <notifications@github.com

**Comment by eddyystop** [comment]
This PR has 2 commits. One is "Change production-ready link from gitbook to quora posting."

Examples fixed with https://github.com/feathersjs/feathers-docs/pull/523.

**Comment by marshallswain** [comment]
I had a chuckle over getting caught doing non-atomic commits / PRs.  I'm super guilty of it.  ;)

**Comment by eddyystop** [comment]
Same here.
